### PR TITLE
Add a FAQ and switch rare issue from README to it

### PR DESCRIPTION
**Pull request by kfox1111**
fixes https://github.com/spiffe/helm-charts/issues/433
closes #385

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Frequently Asked Questions
+
+- [Uninstall is stuck. How do I fix it?](#uninstall-is-stuck-how-do-i-fix-it)
+- [The PSAT plugin is not working](#the-psat-plugin-is-not-working)
+
+## Uninstall is stuck. How do I fix it?
+
+If you uninstall the spiffe csi driver manually before removing the chart, pods can still be using the driver an are unable to mount the csi volume.
```
Comment (marcofranssen) [inline FAQ.md:8]:
```suggestion
If you uninstall the spiffe csi driver manually before removing the chart, pods can still be using the driver an are unable to unmount the csi volume.
```

File: FAQ.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+To resolve, reinstall the chart before trying to remove it again. 
+
+## The PSAT plugin is not working
+
+The chart requires `Projected Service Account Tokens` which has to be enabled on your k8s api server. In most cases this is already done for you.
+
+> **Note** This is enabled by default with newer versions as shown by the existence of:
```
Comment (marcofranssen) [inline FAQ.md:16]:
```suggestion
> **Note**: This is enabled by default with newer versions as shown by the existence of:
```

File: charts/spire/README.md [hunk 3]
```diff
@@ -23,30 +23,6 @@ A Helm chart for deploying the complete Spire stack including: spire-server, spi
 
 > **Note**: For Kubernetes, we will officially support the last 3 versions as described in [k8s versioning](https://kubernetes.io/releases/version-skew-policy/#supported-versions). Any version before the last 3 we will try to support as long it doesn't bring security issues or any big maintenance burden.
```
Comment (marcofranssen) [inline charts/spire/README.md:25]:
```suggestion

## FAQ

For any issues see our [FAQ](../../FAQ.md)…
```

File: charts/spire/README.md.gotmpl [hunk 4]
```diff
@@ -25,30 +25,6 @@
 
 > **Note**: For Kubernetes, we will officially support the last 3 versions as described in [k8s versioning](https://kubernetes.io/releases/version-skew-policy/#supported-versions). Any version before the last 3 we will try to support as long it doesn't bring security issues or any big maintenance burden.
```
Comment (marcofranssen) [inline charts/spire/README.md.gotmpl:27]:
```suggestion

## FAQ
For any issues see our [FAQ](../../FAQ.md)…
```

**Review by marcofranssen — CHANGES_REQUESTED** [review]
Nice start on documenting these things. Few inline suggestions

**Comment by kfox1111** [comment]
Added a faq entry for https://github.com/spiffe/helm-charts/issues/440

File: charts/spire/README.md [hunk 5]
```diff
@@ -23,30 +23,8 @@ A Helm chart for deploying the complete Spire stack including: spire-server, spi
...
-    - command:
-        - kube-apiserver
-        - --api-audiences=api,spire-server
-        - --service-account-issuer=api,spire-agent
-        - --service-account-key-file=/run/config/pki/sa.pub
-        - --service-account-signing-key-file=/run/config/pki/sa.key
-```
-
+## FAQ
+For any issues see our [FAQ](../../FAQ.md)…
```
Comment (marcofranssen) [inline charts/spire/README.md:27]:
```suggestion
For any issues see our [FAQ](../../FAQ.md)…

```

Sorry, my suggestion missed the whiteline 😊

File: charts/spire/README.md.gotmpl [hunk 6]
```diff
@@ -25,30 +25,8 @@
...
-    - command:
-        - kube-apiserver
-        - --api-audiences=api,spire-server
-        - --service-account-issuer=api,spire-agent
-        - --service-account-key-file=/run/config/pki/sa.pub
-        - --service-account-signing-key-file=/run/config/pki/sa.key
-```
-
+## FAQ
+For any issues see our [FAQ](../../FAQ.md)…
```
Comment (marcofranssen) [inline charts/spire/README.md.gotmpl:29]:
```suggestion
For any issues see our [FAQ](../../FAQ.md)…

```

Sorry, my suggestion missed the whiteline 😊

File: FAQ.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+## Uninstall is stuck. How do I fix it?
+
+If you uninstall the spiffe csi driver manually before removing the chart, pods can still be using the driver an are unable to unmount the csi volume.
+
+To resolve, reinstall the chart before trying to remove it again. 
+
+## The PSAT plugin is not working
+
+The chart requires `Projected Service Account Tokens` which has to be enabled on your k8s api server. In most cases this is already done for you.
```
Comment (faisal-memon) [inline FAQ.md:24]:
```suggestion
The chart requires `Projected Service Account Tokens` which has to be enabled on your Kubernetes API server. In most cases this is already done for you.
```

File: FAQ.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
+# Frequently Asked Questions
+
+- [Pods are stuck terminating after uninstall. How do I fix it?](#pods-are-stuck-terminating-after-uninstall-how-do-i-fix-it)
+- [Uninstall is stuck. How do I fix it?](#uninstall-is-stuck-how-do-i-fix-it)
+- [The PSAT plugin is not working](#the-psat-plugin-is-not-working)
+
+## Pods are stuck terminating after uninstall. How do I fix it?
+
+If you uninstall the spire chart before all users of the csi driver are removed, pods will get stuck in a terminating state waiting for the driver, that no longer is installed, to unmount the volumes for the pod. In order to fix this, reinstall the chart and remove all affected workloads that are not part of the spire helm chart itself, before attempting to remove spire again.
```
Comment (faisal-memon) [inline FAQ.md:9]:
```suggestion
If you uninstall the SPIRE chart before all users of the CSI driver are removed, Pods will get stuck in a terminating state waiting for the driver, that no longer is installed, to unmount the volumes for the Pod. In order to fix this, reinstall the chart and remove all affected workloads that are not part of the SPIRE helm chart itself, before attempting to remove SPIRE again.
```

File: FAQ.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+
+You can discover pods that use the driver with the following command:
+```
+kubectl get pods --all-namespaces -o go-template='{{range .items}}{{$nn := printf "%s %s" .metadata.namespace .metadata.name}}{{range .spec.volumes}}{{if .csi.driver}}{{if eq .csi.driver "csi.spiffe.io"}}{{printf "%s\n" $nn}}{{end}}{{end}}{{end}}{{end}}'
+```
+
+## Uninstall is stuck. How do I fix it?
+
+If you uninstall the spiffe csi driver manually before removing the chart, pods can still be using the driver an are unable to unmount the csi volume.
```
Comment (faisal-memon) [inline FAQ.md:18]:
```suggestion
If you uninstall the SPIFFE CSI driver manually before removing the chart, Pods can still be using the driver and are unable to unmount the csi volume.
```

File: FAQ.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+
+- [Pods are stuck terminating after uninstall. How do I fix it?](#pods-are-stuck-terminating-after-uninstall-how-do-i-fix-it)
+- [Uninstall is stuck. How do I fix it?](#uninstall-is-stuck-how-do-i-fix-it)
+- [The PSAT plugin is not working](#the-psat-plugin-is-not-working)
+
+## Pods are stuck terminating after uninstall. How do I fix it?
+
+If you uninstall the SPIRE chart before all users of the CSI driver are removed, Pods will get stuck in a terminating state waiting for the driver, that no longer is installed, to unmount the volumes for the Pod. In order to fix this, reinstall the chart and remove all affected workloads that are not part of the SPIRE helm chart itself, before attempting to remove SPIRE again.
+
+You can discover pods that use the driver with the following command:
```
Comment (faisal-memon) [inline FAQ.md:11]:
```suggestion
You can discover Pods that use the driver with the following command:
```

File: FAQ.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+```
+kubectl get pods --all-namespaces -o go-template='{{range .items}}{{$nn := printf "%s %s" .metadata.namespace .metadata.name}}{{range .spec.volumes}}{{if .csi.driver}}{{if eq .csi.driver "csi.spiffe.io"}}{{printf "%s\n" $nn}}{{end}}{{end}}{{end}}{{end}}'
+```
+
+## Uninstall is stuck. How do I fix it?
+
+If you uninstall the SPIFFE CSI driver manually before removing the chart, Pods can still be using the driver and are unable to unmount the csi volume.
```
Comment (faisal-memon) [inline FAQ.md:18]:
```suggestion
If you uninstall the SPIFFE CSI driver manually before removing the chart, Pods can still be using the driver and are unable to unmount the CSI volume.
```

**Review by faisal-memon — APPROVED** [review]
Thanks @kfox1111 creating this useful FAQ.

**Review by marcofranssen — APPROVED** [review]
LGTM

**Pull request merged**